sup.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from brutal.core.plugin import cmd
+from datetime import datetime
+import admanbot.plugins.calendar_sk as calendar_sk
```

Comment:
Wouldn't it be better to do `from admanbot.plugins import calendar_sk` ?

Also, why `admanbot`?